dna/zomes/projects/src/project/goal/validate.rs

```diff
@@ -43,6 +45,8 @@ pub fn validate_create_entry_goal(
 }
 
 #[hdk_extern]
+/// Updates only allowed if `user_edit_hash` of the updated Goal entry matches the
+/// agent publishing the update
```

Comment:
Can you also add that `user_hash` can't be modified (the address of the "original author" can't change)

Reply:
```suggestion
/// agent publishing the update, `user_hash` cannot change from the original value
```